### ISSUE TITLE
Required changes in css media queries.

### ISSUE THREAD

**Issue by GlitchInMirror** [GlitchInMirror/preview-component#1]
Make the design responsive and to target all devices.

**Comment by artimanputra** [comment]
I would like to work on the responsiveness

**Comment by GlitchInMirror** [comment]
> I would like to work on the responsiveness

Yeah sure go ahead.

**Comment by GlitchInMirror** [comment]
@artimanputra yeah sure go ahead.

**Comment by artimanputra** [comment]
If I send the PR, will it be going to consider in hacktoberfest 2021?

**Comment by GlitchInMirror** [comment]
@artimanputra No, it will not consider. It will consider only when hacktoberfest 2021 started i.e from 01-Oct-21 and u register urself as contributors in the upcoming fest and then raise the pull request.

For any queries join my discord server:
https://discord.gg/9dBS2B58DZ
